lxd-migrate/utils.go

```diff
@@ -44,6 +57,16 @@ func transferRootfs(dst lxd.ContainerServer, op lxd.Operation, rootfs string, rs
 		Fs: &fs,
 	}
 
+	if instanceType == api.InstanceTypeVM {
```

Comment:
Looking at the code, we first do the rsync, then do the block, so the fact that it's reversed here could be the issue.

Reply:
Nevermind